### DISCUSSION TITLE
Is there a general acceptance or resistance to an integration to any blockchain tech?

### DISCUSSION THREAD

**Discussion by jasonsprouse** [mozilla/hubs#5959] [General Questions & Discussion] [answered]
Being that the application runs in the browser it should be possible to easily integrate all the logic necessary to allow in app purchases with any number of blockchain technologies.

**Comment by matthewbcool** [comment, marked as answer]
Since Hubs is a web application and open source, any web standards or technologies you can integrate on the web is a possibility. 

I have seen customized clients use meta mask in the past to handle purchases, you may be able to find some examples searching the Hubs discord. We don't currently have any community guides on [Creator Labs](https://hubs.mozilla.com/labs/) but anyone reading this is welcome to contribute one.

Some examples of customized Hubs code using blockchain related tech that come to mind are HTC's viveverse: https://world.viverse.com/pods/
and Space metaverse: https://www.tryspace.com/
